config/jetpack-cloud-horizon.json

```
@@ -81,6 +81,7 @@
 		"jetpack-cloud-partner-portal": true,
 		"jetpack-cloud-plugin-management": true,
 		"jetpack-cloud-pricing": true,
+		"jetpack-cloud-manage-pricing": false,
```

```suggestion
		"jetpack-cloud-manage-pricing": true,
```
Setting to true based on a PR review suggestion.

---

Thanks Karen!